lld/test/ELF/dependency-file-backslash-as-normal-char.s

```diff
@@ -0,0 +1,13 @@
+# REQUIRES: x86, !system-windows
+# RUN: mkdir -p %t
+# RUN: llvm-mc -filetype=obj -triple=x86_64 %s -o "%t/back\\slash.o"
+# RUN: ld.lld -o %t/foo.exe "%t/back\\slash.o" --dependency-file=%t/foo.d
+# RUN: FileCheck --match-full-lines -DFILE=%t %s < %t/foo.d
+
+# CHECK:      [[FILE]]/foo.exe: \
+# CHECK-NEXT:   [[FILE]]/back\slash.o
```

Comment:
Needs `--match-full-lines --strict-whitespace` to actually test the indentation.